<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="WINDOWS-I42JNHR" failures="1" tests="10" name="Default test" time="30.034" errors="0" timestamp="2024-03-13T18:34:09 IST">
  <testcase classname="com.testscripts.LoginGoogleTests" name="validateDisablemail_google" time="13.643">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at com.pages.LoginWithGoogle.VerifyGooglesignIn(LoginWithGoogle.java:176)
at com.testscripts.LoginGoogleTests.validateDisablemail_google(LoginGoogleTests.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateDisablemail_google -->
  <testcase classname="com.testscripts.LoginGoogleTests" name="ValidateWhatsappConfigEnable" time="0.502"/>
  <testcase classname="com.testscripts.LoginGoogleTests" name="validateTicketForAttendee_google" time="1.051"/>
  <testcase classname="com.testscripts.LoginGoogleTests" name="validateOrderSuccessMessage" time="2.285"/>
  <testcase classname="com.testscripts.LoginGoogleTests" name="validateViewTicketUrl_google" time="1.176"/>
  <testcase classname="com.testscripts.LoginGoogleTests" name="ValidateRegUrl_google" time="2.773"/>
  <testcase classname="com.testscripts.LoginGoogleTests" name="ValidateOrderId_google" time="1.052"/>
  <testcase classname="com.testscripts.LoginGoogleTests" name="validateBuyerDetails_google" time="0.566"/>
  <testcase classname="com.testscripts.LoginGoogleTests" name="validateAttendeeDetails_googleloginViewTickets" time="0.391"/>
  <testcase classname="com.testscripts.LoginGoogleTests" name="validateAttendeeDetails_googleCompleteReg" time="0.721"/>
</testsuite> <!-- Default test -->
